selfdrive/car/hyundai/values.py

```diff
@@ -801,6 +807,13 @@ def match_fw_to_car_fuzzy(live_fw_versions, offline_fw_versions) -> set[str]:
       obd_multiplexing=False,
     ),
   ],
+  # We lose these ECUs without the comma power on these cars.
+  # Note that we still attempt to match with them when they are present
+  non_essential_ecus={
+    Ecu.eps: [CAR.AZERA_6TH_GEN, CAR.AZERA_HEV_6TH_GEN],  # FIXME: EPS responds on a few cars from PT
```

Comment:
wonder if these should be flags? ideally I think we don't reference any particular cars anywhere?

Reply:
This isn't the final form of this, so I'd like to keep it as is since it makes it easier to refactor right now.